packages/frontend-api/src/Model/Customer/User/RegistrationFacade.php

```diff
@@ -58,6 +67,32 @@ public function register(RegistrationData $registrationData): CustomerUser
         return $customerUser;
     }
 
+    /**
+     * @param string $orderUrlHash
+     * @param string $password
+     * @return \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser
+     */
+    public function registerByOrder(string $orderUrlHash, string $password): CustomerUser
+    {
+        $order = $this->orderFacade->getByUrlHashAndDomain($orderUrlHash, $this->domain->getId());
+
+        if ($order->getCustomerUser() !== null) {
+            throw new OrderCannotBePairedException('Order is owned by another customer.');
+        }
+
+        if ($order->getCreatedAt()->getTimestamp() < (time() - self::ONE_HOUR_REGISTRATION_WINDOW)) {
```

Comment:
what about this?
```
$order->getCreatedAt() < new DateTime('-1 hour') 
```

or in seconds using the constant if you want to keep the constant

Reply:
:ok_hand: 43577535a6